src/components/PeopleTable.tsx

```diff
@@ -1,5 +1,49 @@
+import classNames from 'classnames';
+import { Person } from '../types';
+import { useParams, useSearchParams } from 'react-router-dom';
+import React, { useCallback } from 'react';
+import { PersonLink } from './PersonLink';
+import { SearchLink } from './SearchLink';
+
+type PeopleTableProps = {
+  peopleList: Person[];
+};
+
 /* eslint-disable jsx-a11y/control-has-associated-label */
-export const PeopleTable = () => {
+export const PeopleTable: React.FC<PeopleTableProps> = ({ peopleList }) => {
+  const { slug } = useParams();
+  const [searchParams] = useSearchParams();
+
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+
+  const sortParams = (value: string) => {
+    if (sort === value) {
+      return order === 'desc'
+        ? { sort: null, order: null }
+        : { sort: value, order: 'desc' };
```

Comment:
In the `sortParams` function, setting `sort` and `order` to `null` does not remove them from the URL. Consider using `params.delete('key')` to remove these parameters or set them to an empty string if you want to clear them. This will ensure the URL reflects the absence of sorting when the same field is clicked twice.